src/graphql/typeDefs/mutations/communities.graphql

```diff
@@ -0,0 +1,12 @@
+type CommunitiesMutation {
+  "Create a new community"
+  create(community: CommunityCreateInput!): Community!
+    @auth(requires: "communities")
+
+  "delete a community (not implemented)"
+  delete(id: ID!): ID @auth(requires: "communities")
```

Comment:
Not really sure we want to delete a community. Maybe if you're an admin and we just screw up, but a community in theory would have historical value. Maybe it could be deactivated or archived, but I'm not sure about deleted.

Reply:
removed.